lib/Runtime/Language/ModuleNamespace.h

```diff
@@ -61,9 +61,9 @@ namespace Js
         virtual BOOL GetEnumerator(JavascriptStaticEnumerator * enumerator, EnumeratorFlags flags, ScriptContext* requestContext, ForInCache * forInCache = nullptr);
         virtual BOOL SetAccessors(PropertyId propertyId, Var getter, Var setter, PropertyOperationFlags flags = PropertyOperation_None) override { return false; }
         virtual BOOL GetAccessors(PropertyId propertyId, Var *getter, Var *setter, ScriptContext * requestContext) override { return false; }
-        virtual BOOL IsWritable(PropertyId propertyId) override { return false; }
+        virtual BOOL IsWritable(PropertyId propertyId) override { return propertyId != PropertyIds::_symbolToStringTag; }
```

Comment:
Although currently a `ModuleNamespace` object only have `@@toStringTag`, but the `[[Get]]` operation is defined to redirect to `OrdinaryGet` for any symbol.  So these IsWritable/IsEnumerable API should of the same.